src/Microsoft.DocAsCode.Build.JavaScriptReference/BuildInputs/ExceptionInfo.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.JavaScriptReference
+{
+    using System;
+    using YamlDotNet.Serialization;
```

Comment:
>using YamlDotNet.Serialization; [](start = 4, length = 31)

group namespace as usual? `YamlDotNet.Serialization` belongs to third party dependency.